### ISSUE TITLE
Class Base Shopping List's Views

### ISSUE THREAD

**Issue by ghickman** [incuna/dashboard#26]
Add an add/edit view and completely remove the reliance on the admin.

**Comment by mjtamlyn** [comment]
Have started some work on this. Any specific requirements I should know about? What was going on with the added_by stuff in the forms?

**Comment by ghickman** [comment]
Not really, the whole app is pretty much the generic cbvs. Might as well get rid of the admin access for normal users in favour of showcasing the cbvs. I think moving away from the admin in general is probably for the best. Might be able to expose an API/text gateway for this app later then.

added_by is more IMC stuff - related to changes made for #8. Should be fixed in develop and feature-imc now.
